packages/widget/src/i18n/en.json

```diff
@@ -259,5 +259,11 @@
   },
   "wallet": {
     "extensionNotFound": "Please make sure that only the {{name}} browser extension is active before choosing this wallet."
+  },
+  "multisig": {
+    "alert": {
+      "title": "Multisig wallet connected",
+      "description": "Destination wallet address is required to complete the exchange."
+    }
```

Comment:
Shouldn't be here, move to Jumper, please.

Reply:
Checking